src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -615,3 +615,122 @@ app.use(function processItems(req, res, next) {
   })
 });
 ```
+
+### Vue.js
+
+The Vue Tracing Integration allows us to track rendering performance during an initial application load.
+
+It is achieved by injecting our handler inside Vue's `beforeCreate` mixin, which gives us access to a Vue component during all of it's lifecycle stages.
+When we encounter component named `root`, which is a top-level Vue instance (as in `new Vue({})`), we use our AM Tracing integration,
+and create a new activity named `Vue Application Render`. Once it is created, it will wait until all of it's child components are rendered, and there are no new rendering events triggered within configured `timeout`, before marking the activity as completed.
```

Comment:
```suggestion
and create a new activity named `Vue Application Render`. Once the activity has been created, it will wait until all of its child components render, and there aren't new rendering events triggered within the configured `timeout`, before marking the activity as completed.
```